From c10bc61508dc52b8315084e628f36a6c3c2dabb1 Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Sun, 10 Sep 2017 13:34:54 +0200
Subject: [PATCH 28/30] powerpc{64}: fix MAP_NORESERVE and MAP_LOCKED in mman.h

MAP_{NORESERVE,LOCKED} have different values on powerpc than in generic.
---
 arch/powerpc/bits/mman.h   | 7 ++++++-
 arch/powerpc64/bits/mman.h | 5 +++++
 2 files changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/powerpc/bits/mman.h b/arch/powerpc/bits/mman.h
index b6a15a12..95ec4358 100644
--- a/arch/powerpc/bits/mman.h
+++ b/arch/powerpc/bits/mman.h
@@ -1,4 +1,9 @@
-#define	PROT_SAO       0x10
+#define PROT_SAO       0x10
+
+#undef MAP_NORESERVE
+#define MAP_NORESERVE   0x40
+#undef MAP_LOCKED
+#define MAP_LOCKED	0x80
 
 #undef MCL_CURRENT
 #define MCL_CURRENT     0x2000
diff --git a/arch/powerpc64/bits/mman.h b/arch/powerpc64/bits/mman.h
index f5974f82..95ec4358 100644
--- a/arch/powerpc64/bits/mman.h
+++ b/arch/powerpc64/bits/mman.h
@@ -1,5 +1,10 @@
 #define PROT_SAO       0x10
 
+#undef MAP_NORESERVE
+#define MAP_NORESERVE   0x40
+#undef MAP_LOCKED
+#define MAP_LOCKED	0x80
+
 #undef MCL_CURRENT
 #define MCL_CURRENT     0x2000
 #undef MCL_FUTURE
-- 
2.13.3

